Rubberduck.Parsing/VBA/ReferencePriorityMap.cs

```diff
@@ -8,16 +8,16 @@ namespace Rubberduck.Parsing.VBA
     /// </summary>
     public class ReferencePriorityMap : Dictionary<string, int>
     {
-        private readonly string _referenceId;
+        private readonly string _referencedProjectId;
 
-        public ReferencePriorityMap(string referenceId)
+        public ReferencePriorityMap(string referencedProjectId)
         {
-            _referenceId = referenceId;
+            _referencedProjectId = referencedProjectId;
         }
 
-        public string ReferenceId
+        public string ReferencedProjectId
         {
-            get { return _referenceId; }
+            get { return _referencedProjectId; }
```

Comment:
So, do I read this correctly as *it will also work for referenced VBA projects*?